src/parquet/file/file-serialize-test.cc

```diff
@@ -97,6 +102,18 @@ TEST_F(TestSerialize, SmallFile) {
   ASSERT_EQ(values, values_out);
 }
 
+TEST_F(TestSerialize, SmallFileUncompressed) {
+  FileSerializeTest(Compression::UNCOMPRESSED, node_);
```

Comment:
Ah, I see the problem. You need to make FileSerialzeTest a member function of the TestSerialze class.